engine/src/cmd/collision.cpp

```diff
@@ -292,15 +292,24 @@ void Collision::collide( Unit* unit1,
     // See commit a66bdcfa1e00bf039183603913567d48e52c7a8e method Unit->jumpReactToCollision for an example
     // I assume this is for "always open" jump gates that you pass through
 
+    collision1.apply_force = false;
+    collision1.deal_damage = false;
+    collision2.apply_force = false;
+    collision2.deal_damage = false;
+
     collision1.shouldApplyForceAndDealDamage(unit2);
     collision2.shouldApplyForceAndDealDamage(unit1);
 
     float elasticity = 0.8f;
-    collision1.applyForce(elasticity, collision2.mass, collision2.velocity);
-    collision2.applyForce(elasticity, collision1.mass, collision1.velocity);
+    if (collision1.apply_force && collision2.apply_force) {
```

Comment:
Also not needed. First line of applyForce function returns if false.

Reply:
The behavior definitely changed when I made this change to the code. The difference is that now, no force is applied to **either** object/unit unless **both** pass the "shouldApplyForce" test. Whereas before, the conditions applied to the units individually.

Reply:
OK, I went ahead and undid this. It didn't turn out to be necessary after all.